src/core/tracking.js

```diff
@@ -159,9 +171,34 @@ export function getAction(type) {
   return (
     {
       [ADDON_TYPE_EXTENSION]: TRACKING_TYPE_EXTENSION,
+      [ADDON_TYPE_OPENSEARCH]: TRACKING_TYPE_EXTENSION,
+      [ADDON_TYPE_STATIC_THEME]: TRACKING_TYPE_STATIC_THEME,
       [ADDON_TYPE_THEME]: TRACKING_TYPE_THEME,
     }[type] || TRACKING_TYPE_INVALID
   );
 }
 
+export const getAddonEventCategory = (type, installAction) => {
+  let category;
```

Comment:
Instead of defining a `category` here, populating it in the switch, and returning it at the end, could you just return the category from inside the switch?  Something like this:

```js
export const getAddonEventCategory = (type, installAction) => {
  const isTheme = ADDON_TYPE_THEMES.includes(type);

  switch (installAction) {
    case INSTALL_STARTED_ACTION:
      return isTheme
        ? INSTALL_THEME_STARTED_CATEGORY
        : INSTALL_EXTENSION_STARTED_CATEGORY;
    case UNINSTALL_ACTION:
      return isTheme
        ? UNINSTALL_THEME_CATEGORY
        : UNINSTALL_EXTENSION_CATEGORY;
    default:
      return isTheme ? INSTALL_THEME_CATEGORY : INSTALL_EXTENSION_CATEGORY;
  }
};
```